src/Symfony/Component/DependencyInjection/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ CHANGELOG
 3.4.0
 -----
 
+ * moved the `ExtensionCompilerPass` to before-optimization passes with priority -1000
```

Comment:
may be worth adding this to the upgrade file too

Reply:
done